diff --git a/node_modules/@pricava/react-native-google-credential/android/build.gradle b/node_modules/@pricava/react-native-google-credential/android/build.gradle
index ab2828a..b6844b0 100644
--- a/node_modules/@pricava/react-native-google-credential/android/build.gradle
+++ b/node_modules/@pricava/react-native-google-credential/android/build.gradle
@@ -10,9 +10,12 @@ version = '0.1.0'
 android {
   namespace "com.pricava.googlecredential"
 
+  compileSdk rootProject.ext.compileSdkVersion
+
   defaultConfig {
     versionCode 1
     versionName "0.1.0"
+    minSdkVersion rootProject.ext.minSdkVersion
   }
 }
 
